src/mame/drivers/hp9k_3xx.cpp

```diff
@@ -185,6 +196,9 @@ class hp9k3xx_state : public driver_device
 	uint8_t m_latch_data;
 	uint32_t m_lastpc;
 	bool gpib_irq_line;
+
+	uint8_t m_rtc_data;
+	bool m_old_latch_enable;
```

Comment:
You really should be registering stuff for save states - it makes your life easier, too, because you can save/restore state at a problematic point to aid in debugging.